.travis.yml

```diff
@@ -2,7 +2,9 @@ language: java
 sudo: false # faster builds
 
 jdk:
+  - openjdk11
```

Comment:
Why do we need both? I think we only need to choose between oraclejdk and openjdk.

Reply:
I have the same question.